### DISCUSSION TITLE
How do I find/choose/name the vacuum entity?

### DISCUSSION THREAD

**Discussion by Ted-Corrrell** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#290] [Q&A] [answered]
When I copied parts of the demo_config.yaml to the card I was trying to set up, it gives me the message "Invalid Entities: vacuum.xiaomi_vacuum". I tried the host ip address from the xiaomi_cloud_map_extractor. It said that was invalid too. What do I need to put as the entity?

**Comment by PiotrMachowski** [comment, marked as answer]
Here you can find all entity ids:
[![Open your Home Assistant instance and show your state developer tools.](https://my.home-assistant.io/badges/developer_states.svg)](https://my.home-assistant.io/redirect/developer_states/)

**Reply by Ted-Corrrell** [reply]
Turns out I missed the part where I needed the built-in Xiaomi integration. Once I installed that, there was a drop down in the UI editor that had my vacuum. I already had parts of the demo config in the code editor. The UI editor put my vacuum into the entity line of the existing YAML and it worked instantly. 
Thank you for a quick reply & for a great card!
